src/debugAdapter/goDebug.ts

```diff
@@ -743,14 +744,34 @@ class GoDebugSession extends LoggingDebugSession {
 			return breakpoints;
 		}).then(breakpoints => {
 			response.body = { breakpoints };
-			this.sendResponse(response);
 			log('SetBreakPointsResponse');
+			return this.sendResponse(response);
 		}, err => {
-			this.sendErrorResponse(response, 2002, 'Failed to set breakpoint: "{e}"', { e: err.toString() });
 			logError(err);
+			return this.sendErrorResponse(response, 2002, 'Failed to set breakpoint: "{e}"', { e: err.toString() });
```

Comment:
Any reason for the `return` here and in line 748?
Both `sendResponse` and `sendErrorResponse` return void.

Also, whats the reason for the swapping of lines?

Reply:
Reverted